rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -50,6 +50,7 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
 RUN R -e "install.packages('Rcpp')"
+RUN if [ -n "$NEEDPACKAGE" ]; then R -e "install.packages(c('tidyverse', 'tidymodels', 'plumber', 'vetiver', 'devtools'))" ;  echo "The variable NEEDPACKAGE not set -> skipping installation of extra R packages"; fi
```

Comment:
Also the versions are gone now. It was good that we tried to pin to particular version. I was just pointing out that it was a bit blurry to me. But maybe versions are not part of this task? :slightly_smiling_face:

Reply:
Thanks. Resolving this for now.